admin/billing/orb.go

```diff
@@ -0,0 +1,422 @@
+package billing
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/eapache/go-resiliency/retrier"
+	"github.com/orbcorp/orb-go"
+	"github.com/orbcorp/orb-go/option"
+	"github.com/rilldata/rill/admin/database"
+)
+
+const (
+	requestMaxLimit = 500
+	requestTimeout  = 10 * time.Second
+)
+
+var ErrNotFound = errors.New("not found")
```

Comment:
nit: Move to `biller.go` since this in theory could be returned by multiple billing implementations